app/models/check/api.rb

```diff
@@ -0,0 +1,10 @@
+module Check
```

Comment:
`Check` is too generic. How about `application_health_check`? Maybe put `Check` under `ApplicationHealth` module?